cluster/sbin/collect-debug-artifacts.sh

```diff
@@ -88,6 +101,28 @@ function check_configs {
     echo NUM_STACK_DUMPS=$NO_OF_STACK_DUMPS
     echo INTERVAL_BETWEEN_DUMPS=$INTERVAL_BETWEEN_DUMPS
     echo GET_EVERYTHING=${GET_EVERYTHING}
+    echo START TIME = "${START_TIME}"
+    echo END   TIME = "${END_TIME}"
+  fi
+
+  if [ -z "${START_TIME}" ]; then
+    START_EPOCH=0
+  else
+    START_EPOCH=$(date +%s --date "${START_TIME}" 2>/dev/null)
+    if ! [[ "$START_EPOCH" =~ $num_regex ]] ; then
+      echo "${START_TIME} error: Not expected date format" 
```

Comment:
Error message will seem a bit odd with time at the start like: `Thu 5 05:36:04 error: Not expected date format`
Instead something like `echo "Error: not expected date format '${START_TIME}'"